src/components/containers/ScrollView.tsx

```diff
@@ -79,7 +78,10 @@ const ScrollViewWithHeadersInputComp = (
         ref={scrollRef}
         scrollEventThrottle={16}
         overScrollMode="auto"
-        onScroll={scrollHandler}
+        onScroll={(e) => {
+          scrollHandler(e);
```

Comment:
From my testing on the `example` application, this code does not work. We cannot simply call the scrollHandler function this way since it is running in a different context. Have you checked to see if the code changes work?

I tried to look into passing multiple handlers to a single `onScroll` value, but [reanimated does not seem to support that at the moment](https://github.com/software-mansion/react-native-reanimated/discussions/1763).

I think the only solution right now, even though it is somewhat ugly, is to allow `worklets` to be passed directly and added to the existing scroll handler that is used internally.